deploy/ocp/manifests/0.16.0/0000_50_olm_00-catalogsources.crd.yaml

```diff
@@ -0,0 +1,201 @@
+---
```

Comment:
Please drop the changes to anything other than the `/manifests` directory for this patch.

Reply:
ok, will update the PR. Thanks!